packages/flutter_tools/lib/src/asset.dart

```diff
@@ -820,10 +821,9 @@ class ManifestAssetBundle implements AssetBundle {
       return;
     }
 
-    final Iterable<File> files = _fileSystem
-      .directory(directoryPath)
-      .listSync()
-      .whereType<File>();
+    final Iterable<FileSystemEntity> entities = _fileSystem.directory(directoryPath).listSync();
+
```

Comment:
```suggestion
```

Reply:
I like the blank like between `entities` and `files` because it puts some distance between the `for` loop that iterates over the files. The reason I like this separation is that `entities` is not only used for the file `for` loop--it's also used for the `for` loop.